README.md

```diff
@@ -56,6 +56,7 @@ To build LynxKite you will need:
 - Java 8 and [SBT](https://www.scala-sbt.org/)
 - Python 3.7+ with [Conda](https://docs.conda.io/en/latest/miniconda.html)
 - Go 1.14+
+- Swig 4.0+
```

Comment:
Thanks! You also need `cmake` now because it's used by NetworKit.

Reply:
Done